components/infobox/wikis/leagueoflegends/infobox_team_custom.lua

```diff
@@ -0,0 +1,75 @@
+---
+-- @Liquipedia
+-- wiki=leagueoflegends
+-- page=Module:Infobox/Team/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Team = require('Module:Infobox/Team')
+local Variables = require('Module:Variables')
+local Class = require('Module:Class')
+local String = require('Module:StringUtils')
+local Template = require('Module:Template')
+
+local CustomTeam = Class.new()
+
+local _team
+
+function CustomTeam.run(frame)
+	local team = Team(frame)
+	_team = team
+	team.createBottomContent = CustomTeam.createBottomContent
+	team.addToLpdb = CustomTeam.addToLpdb
+	team.getWikiCategories = CustomTeam.getWikiCategories
+	return team:createInfobox(frame)
+end
+
+function CustomTeam:createBottomContent()
+	return Template.expandTemplate(
+		mw.getCurrentFrame(),
+		'Upcoming and ongoing matches of',
+		{team = _team.name}
+	) .. Template.expandTemplate(
+		mw.getCurrentFrame(),
+		'Upcoming and ongoing tournaments of',
+		{team = _team.name}
+	)
+end
+
+function CustomTeam:addToLpdb(lpdbData, args)
+	mw.logObject(args)
+	if not String.isEmpty(args.teamcardimage) then
+		lpdbData.logo = args.teamcardimage
+	elseif not String.isEmpty(args.image) then
+		lpdbData.logo = args.image
+	end
+
+	lpdbData.region = Variables.varDefault('region', '')
+
+	lpdbData.extradata = {
```

Comment:
current lol infobox doesn't have the args you use in this extradata block —> are they intentional new params or is it a mistake?
this also overwrites any data set into extradata by commons, e.g. earnings per year data